helix-rest/src/test/java/org/apache/helix/rest/server/TestClusterAccessor.java

```diff
@@ -405,7 +407,68 @@ public void testEnableDisableMaintenanceModeWithCustomFields() {
         accessor.getBaseDataAccessor().exists(accessor.keyBuilder().maintenance().getPath(), 0));
   }
 
-  @Test()
+  @Test(dependsOnMethods = "testEnableDisableMaintenanceModeWithCustomFields")
+  public void testGetStateModelDef() throws IOException {
+
+    System.out.println("Start test :" + TestHelper.getTestMethodName());
+    String cluster = "TestCluster_1";
+    String stateModelDefs = get("clusters/" + cluster + "/statemodeldefs", null, Response.Status.OK.getStatusCode(), true);
```

Comment:
It seems you are using multiple times for "clusters/" + cluster + "/statemodeldefs/". Would it be better to declare a var like clusterStateModelDefUri = "clusters/" + cluster + "/statemodeldefs/": avoid creating multiple string objects in Java.

Reply:
I think you are right that string is immutable and using stringbuilder instead of using + is a better approach. This just follows the style of other code in the test case. Also, this kind of performance lose will not be a problem for test cases with a tradeoff of better readibility. That said, I will try to use a variable here.

Reply:
+ actually calls StringBuilder() in JDK like:
new StringBuilder().append(str1).append(str2).toString();
So your code is fine in this case. I am sorry about the confusion.